README.rst

```diff
@@ -30,27 +30,27 @@ You're all set! Retrieve the list of available indices by running in your termin
 
 ::
 
-    run-func search getIndices
+    run-func index.js getExistingIndices
 
 
 How to use
 ----------
 
-1. Make sure that you have `full_format_recipes.json` at the root level of the repository and run command to inject data::
+1. Make sure that you have `full_format_recipes.json` at the root level of the repository and run command to index data:
 
-    run-func index injectData
+    run-func index.js indexData
 
 Wait till the data is indexed. Can take 15-20 seconds.
 
 2. Check `search.js` and `aggregate.js` for examples you can run. Every method contains a terminal command sample that you can try.
 
 ::
 
-    run-func search match title "garlic onion cake"
+    run-func search.js match title "garlic onion cake"
 
 ::
 
-    run-func aggregate field rating
+    run-func aggregate.js field averageRating
```

Comment:
See https://github.com/Aiven-Labs/demo-open-search-node-js/blob/main/aggregate.js#L8

Reply:
```
; run-func aggregate.js field averageRating

/opt/homebrew/lib/node_modules/run-func/index.js:50
    throw new Error(`Function ${fnName} is not present or exported from module`);
          ^

Error: Function field is not present or exported from module
    at executeInModule (/opt/homebrew/lib/node_modules/run-func/index.js:50:11)
    at Object.<anonymous> (/opt/homebrew/lib/node_modules/run-func/index.js:37:3)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Module._load (node:internal/modules/cjs/loader:1023:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)
    at node:internal/main/run_main_module:28:49

Node.js v21.1.0
```

Reply:
But the following works:
```
; run-func aggregate.js averageRating

{ value: 3.7130597014925373 }
```